stories/ui/Badge.js

```diff
@@ -6,16 +6,34 @@ import Badge from 'ui/components/Badge';
 
 const label = 'Hello Badge';
 
-storiesOf('Badge', module)
-  .add('default (no type)', () => <Badge label={label} />)
-  .add('experimental type', () => <Badge type="experimental" label={label} />)
-  .add('featured type', () => <Badge type="featured" label={label} />)
-  .add('restart-required type', () => (
-    <Badge type="restart-required" label={label} />
-  ))
-  .add('not-compatible type', () => (
-    <Badge type="not-compatible" label={label} />
-  ))
-  .add('requires-payment type', () => (
-    <Badge type="requires-payment" label={label} />
-  ));
+const sections = [
+  {
+    type: 'featured',
+  },
+  {
+    type: 'experimental',
+  },
+  {
+    type: 'restart-required',
+  },
+  {
+    type: 'not-compatible',
+  },
+  {
+    type: 'requires-payment',
+  },
+];
+
+storiesOf('Badge', module).addWithChapters('Badge variations', {
+  useTheme: true,
```

Comment:
Rather than set this to `false` for every component, it looks like you can just set it as a default (according to the docs):

```diff
diff --git a/stories/setup/config.js b/stories/setup/config.js
index eadb40aea..a4a8ad884 100644
--- a/stories/setup/config.js
+++ b/stories/setup/config.js
@@ -9,6 +9,7 @@ setDefaults({
     allowSourceToggling: false,
     showPropTables: false,
     allowPropTablesToggling: false,
+    useTheme: false,
   },
 });
 
```

However, I don't see a visual difference by changing this parameter in either case.

Reply:
It seems to be a minimal difference as far as I can see. But there are some inline styles that are added so yes, I will make sure to set this to false. As noted above it should eliminate the !importants added to some chapter styles that I set up.